common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -88,3 +92,153 @@ def test_page_has_accomplishments_banner(self):
             }
         ]
         self.assert_events_match(expected_events, actual_events)
+
+
+@attr('shard_5')
+class CertificateProgressPageTest(EventsTestMixin, UniqueCourseTest):
```

Comment:
I don't think we need to inherit from `EventsTestMixin` since we are not testing any event tracking.